config/crd/bases/v1/datadoghq.com_datadogpodautoscalers_v2alpha1.json

```diff
@@ -0,0 +1,749 @@
+{
+  "additionalProperties": false,
```

Comment:
Leftover from v2alpha1 version, should be deleted.

Reply:
Still pending deletion!

Reply:
done